language-support/hs/bindings/examples/View.hs

```diff
@@ -1,15 +1,40 @@
 -- Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 -- SPDX-License-Identifier: Apache-2.0
 
+{-# LANGUAGE OverloadedStrings #-}
+
 module Main(main) where
 
-import           LedgerIdentity     (Port (..), ledgerId)
+import           Control.Concurrent
+import           LedgerHL           as Ledger
 import           System.Environment
 
 main :: IO ()
 main = do
-    putStrLn "Ledger App (View)..."
-    [port] <- getArgs
-    id <- LedgerIdentity.ledgerId (Port (read port))
-    print (port, id)
+    port <- fmap (Port . read . head) $ getArgs
+    putStrLn$ "Connecting to ledger on port: " ++ show port
```

Comment:
```suggestion
    putStrLn $ "Connecting to ledger on port: " ++ show port
```

Everyone always puts spaces before and after `$`. Your `putStrLn$` gives me flash backs to visual basic.

Reply:
More generally, we hashed it out and advocate white space around all binary operators, notably including cons e.g `x : xs`.

Reply:
ok